unreleased.rst

```diff
@@ -22,3 +22,5 @@ HEAD — ongoing
 - [DAML Compiler] **BREAKING CHANGE**: Drop support for DAML-LF 1.4. Compiling to DAML-LF 1.5 should work without any code changes, although we highly recommend not specifying a target DAML-LF version at all. (The ledger server still supports DAML-LF 1.4.)
 - [Sandbox] Made the archive CLI arguments optional. 
   See `#1905 <https://github.com/digital-asset/daml/issues/1905>`__.
+- [Ledger API] **BREAKING CHANGE**: Specify the C# namespace in the ledger api proto
```

Comment:
Why is that a breaking change? You mean because C# bindings generated now would end up in a different namespace than the default one?

Reply:
as you don't provide any C# bindings at present then I would have thought this is moot. I suggest that not many people are using C# at the moment as otherwise they would have faced the None enum issue that is referenced in #1900, so I would force the 'breaking change' :-)

Reply:
Exactly. I added a sentence.